docs/contributing/hacking.rst

```diff
@@ -39,7 +39,7 @@ At a glance, you will need:
 - VirtualBox latest
 - Vagrant 1.5 or later
 - On Mac, you will also want
-  - Boot2Docker
+  - Docker Toolbox (https://www.docker.com/toolbox), which provides Docker Machine.
```

Comment:
I'd just say `docker` and on mac `docker-machine`.

Docker is a dependency on linux as well, and I, for one, use `homebrew` to install docker components, so I like to know which specific components I need.